jina/serve/runtimes/gateway/http/models.py

```diff
@@ -238,7 +239,36 @@ def _get_example_data():
     return docs.to_list()
 
 
-class JinaRequestModel(BaseModel):
+class PydanticToDictModel(BaseModel):
+    """
+    Base class for models that contain PydanticDocument or PydanticDocarray and need to be converted to dict.
+    """
+
+    def dict(self, *args, **kwargs):
+        """
+        Generate a dictionary representation of the model, optionally specifying which fields to include or exclude.
```

Comment:
not sure to see in the code how to specify which field to include/exclude is it in the super().dict() method ?

Reply:
Exactly, the docstring is just copied from the same method in `super()`. This method itself just replaces one filed in the dict provided by super